.gitignore

```diff
@@ -1,3 +1,61 @@
+# https://github.com/github/gitignore/blob/main/Autotools.gitignore
+# http://www.gnu.org/software/automake
+
+Makefile.in
+/ar-lib
+/mdate-sh
+/py-compile
+/test-driver
+/ylwrap
+.deps/
+.dirstamp
+
+# http://www.gnu.org/software/autoconf
+
+autom4te.cache
+/autoscan.log
+/autoscan-*.log
+/aclocal.m4
+/compile
+/config.cache
+/config.guess
+/config.h.in
+/config.log
+/config.status
+/config.sub
+/configure
```

Comment:
I think i would keep configure in git for convenience

Reply:
`configure` file is machine-dependent (e.g. latest autoconf version can differ in different platforms), and it has to be updated manually on git. If someone forgets to update ./configure with autogen.sh and only push changes on autogen.sh, this might cause inconvinience for other devs, too.

If there is a way to always keep `configure` up-to-date, I will leave it out from .gitignore